data/object-key-short-code-ii.js

```diff
@@ -1 +1 @@

```

Comment:
Aw, shoot! Not the right object! (problem fixed by another commit to master branch)